### PR TITLE
[WIP] Changes as suggested during PR review

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

File: tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "portMappings": [
         {
           "containerPort": 80,
-          "servicePort": $hostport,
```
Comment (kadel) [inline tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json]: see https://github.com/projectatomic/atomicapp/pull/392#discussion_r46400300 
sorry :-(

Comment (dharmit) [inline tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json]: Probably, I went too early into changing it. My bad. :disappointed:

Comment (kadel) [inline tests/units/cli/test_examples/helloapache/artifacts/marathon/helloapache.json]: I just tested it to be sure, and it really needs to be without quotes or you get this
`{"message":"Invalid JSON","details":[{"path":"/container/docker/portMappings(0)/servicePort","errors":["error.expected.jsnumber"]}]}`

**Comment by dharmit** [comment]
@kadel I've made some changes. Take a look and let me know your thoughts.

**Pull request merged**